tests/OmniSharp.Roslyn.CSharp.Tests/SemanticHighlightFacts.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.Models.SemanticHighlight;
+using OmniSharp.Models.V2;
+using OmniSharp.Roslyn.CSharp.Services.SemanticHighlight;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Roslyn.CSharp.Tests
+{
+    public class SemanticHighlightFacts : AbstractSingleRequestHandlerTestFixture<SemanticHighlightService>
```

Comment:
can we have one more test with a static that checks that `AdditiveTypeNames` have been excluded?

Reply:
Better yet. I added a test to ensure it gets used =)